numpy/core/multiarray.py

```diff
@@ -23,6 +23,7 @@
     'MAY_SHARE_BOUNDS', 'MAY_SHARE_EXACT', 'NEEDS_INIT', 'NEEDS_PYAPI',
     'RAISE', 'USE_GETITEM', 'USE_SETITEM', 'WRAP', '_fastCopyAndTranspose',
     '_flagdict', '_insert', '_reconstruct', '_vec_string', '_monotonicity',
+    '_get_ndarray_c_version',
```

Comment:
Why add this one?

Reply:
It was imported on line 17 along with other members from `_multiarray_umath` (from `multiarraymodule.c`), but not used elswhere in this file. It seems that it should have been included here too, but left out. This function is documented in `numpy/core/_add_newdocs.py`.

Reply:
Strange, I would expect this addition to fail the API tests in `numpy/tests/test_public_api.py`. @rgommers ideas?

Reply:
We could split this oddity off as separate issue to track down. If someone knows how to phrase this issue, please do, otherwise I'll make an attempt later this week.

Reply:
xref gh-15518, so it can be removed from this PR